CLAUDE.md

```
@@ -0,0 +1,153 @@
+# Pre-commit check
+
+Before every commit, run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)